cmd/cephcsi.go

```diff
@@ -27,7 +27,7 @@ import (
 	"github.com/ceph/ceph-csi/internal/controller"
 	"github.com/ceph/ceph-csi/internal/controller/persistentvolume"
 	"github.com/ceph/ceph-csi/internal/liveness"
-	"github.com/ceph/ceph-csi/internal/rbd"
+	rbddriver "github.com/ceph/ceph-csi/internal/rbd-driver"
```

Comment:
forgot to post this comment in the last review.

can this be `github.com/ceph/ceph-csi/internal/rbd/driver` IMO moving just driver .go to a new folder outside or rbd packages don't look good?

Reply:
sure, that should be possible too